conda/recipes/rapids_core_dependencies/build.sh

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+
+# Script assumes the script is executed from the root of the repo directory
+
+REPO_DIR=$(cd $(dirname $0); pwd)
+
+SOURCE_DIR=${REPO_DIR}/conda/recipes/rapids_core_dependencies
+BUILD_DIR=${BUILD_DIR:=${REPO_DIR}/build}
+
+INSTALL_PREFIX=${INSTALL_PREFIX:=${PREFIX:=${CONDA_PREFIX:=$BUILD_DIR/install}}}
+export PARALLEL_LEVEL=${PARALLEL_LEVEL:-4}
+
+for bd in ${BUILD_DIRS}; do
+  if [ -d "${bd}" ]; then
+      find "${bd}" -mindepth 1 -delete
+      rmdir "${bd}" || true
```

Comment:
How does this compare to `rm -rf`?

Reply:
I lifted this from rmm / cudf and how the `build.sh --clean` operation works ( https://github.com/rapidsai/rmm/blob/branch-22.12/build.sh#L150 ).

I will update the script to document the why of this over `rm -rf`